packages/mobile/ios/Quiet/Info.plist

```diff
@@ -39,6 +39,8 @@
 	<false/>
```

Comment:
All of the items added here, even the ones that don't seem relevant (e.g. `NSAppleMusicUsageDescription`) are required to get access to files we need for uploading files.